### DISCUSSION TITLE
Welcome to ipsw Discussions!

### DISCUSSION THREAD

**Discussion by blacktop** [blacktop/ipsw#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TurtleToo** [comment]
Hello all 😊 
For several months my iOS
![IMG_0456](https://github.com/blacktop/ipsw/assets/40183659/bc9c886c-bba9-43ce-977e-acc4e9d149ca)
 device has been crashing every time I get a message, notification, and others. I’ve been to apple support, geek squad, researches on my own to find the answer as to why 1/2 my disks on the phone are cleaned into the negative. For the love of cotton socks, help!
![IMG_0447](https://github.com/blacktop/ipsw/assets/40183659/eb804807-f234-4370-bc45-ac7f70e079b0)
![IMG_0448](https://github.com/blacktop/ipsw/assets/40183659/d7d55b44-156e-46ee-8e7c-83c1cd55ff5a)
![IMG_0449](https://github.com/blacktop/ipsw/assets/40183659/421e401d-3b7b-4016-be1f-46762a56e48d)
![RPReplay_Final1700529247-compressed](https://github.com/blacktop/ipsw/assets/40183659/674a56b6-7fe5-498f-81c0-b2f655dc0c2f)
